test/conformance/kuttl/generate/clusterpolicy/cornercases/pod-restart-on-cm-update/04-update-sc.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kube-state-metrics-crds
+  namespace: kube-state-metrics
+stringData:
+  foo: not-bar
```

Comment:
I'm not sure if you can do this. Using `stringData` requires that the Secret controller encode to base64 prior to persisting. It doesn't get persisted as plain text.

Reply:
So, when I gave `foo: bar` in `data` it didn't work, and required that I should convert it to `base64`. But I looked up, and found that you can give it directly in `stringData` without encoding. That's why I did this.

Should I change it to `foo: YmFy` in `data`?

Reply:
>So, when I gave foo: bar in data it didn't work
That's correct because contents under `data` are expected to be base64-encoded. When you use `stringData` this is just a measure of convenience; the Secret is still persisted with `data` and its contents in base64. Use of `stringData` is only ever appropriate upon the initial creation of a Secret, not subsequent updates.

Reply:
Alright, changing it to `foo: YmFy`